src/plugins/blockresolver/README.md

```diff
@@ -31,43 +32,43 @@ A block consists of 2 parts:
 Currently the identifier must be unique.
  
 ## Example ##
-```
```

Comment:
Is this done by hand or with tool?

If you have a tool, we can run it on the build server (similar to clang-format)